include/SugarFields/Fields/Currency/SugarFieldCurrency.php

```diff
@@ -95,7 +95,14 @@ public function importSanitize(
         ) {
         $value = str_replace($settings->currency_symbol, "", $value);
         
-        return $settings->float($value, $vardef, $focus);
+        $value = str_replace($settings->num_grp_sep, "", $value);       
+
+        if (isset($vardef['len'])) {
+            // check for field length
+            $value = sugar_substr($value, $vardef['len']);
+        }
+
+        return $value;
```

Comment:
Hi @SinergiaCRM,

Thank you for your PR.

Don't we also want to make sure the decimal separator is a "."? or is that what causes the problem on currencies at the moment?

Reply:
Hi @SinergiaCRM, thank you. Makes sense.